src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/ManifestTest.cs

```diff
@@ -26,8 +26,8 @@ public partial class ManifestTest : BaseTest
 	<application android:label=""CheckElementReOrdering"">
 	</application>
 	<permission android:name=""com.xamarin.test.TEST"" android:label=""Test Permission"" />
-	<permissionTree android:name=""com.xamarin.test"" />
-	<permissionGroup android:name=""group1"" />
+	<permission-tree android:name=""com.xamarin.test"" />
```

Comment:
How was this not a problem before?! :-/

Reply:
I guess because `aapt` was ok with the old `permissionTree`, but the new `aapt2` is not. It reports them as invalid. 
I suspect google changed it at some point. not sure when. The old style does not even appear in the docs

[1] https://developer.android.com/guide/topics/manifest/permission-tree-element